### PR TITLE
Eliminates "term" adds examples

### PR DESCRIPTION
**Pull request by emileighoutlaw**
Per: https://github.com/18F/fec-cms/issues/531

I'm not 100% confident in the HTML and styling here, so cc @noahmanger @jenniferthibault

### CODE REVIEW

**Comment by emileighoutlaw** [comment]
Ping! This one has been open for a while — anyone want to merge?

@noahmanger @xtine

**Comment by emileighoutlaw** [comment]
Can we squeeze this one in today before the holiday? @xtine @noahmanger

**Comment by emileighoutlaw** [comment]
Removed the class

File: fec_eregs/templates/regulations/generic_universal.html [hunk 1]
```diff
@@ -14,11 +14,12 @@
           <div class="search-header">Search regulations</div>
           <form action="{{ FEC_WEB_URL }}/legal/search/regulations/" method="GET" role="search">
             <div class="combo--search--mini">
-              <label class="label t-left-aligned" for="search-regs">Terms</label>
+              <label class="label t-left-aligned" for="search-regs">Keywords</label>
               <input id="search-regs" name="search" class="combo__input" type="text" title="Search term"/>
               <input type="hidden" name="search_type" id="version-hidden" title="Search Type" value="regulations"/>
               <button class="combo__button button--search button--standard" type="submit"/>
                 <span class="u-visually-hidden">Search</span></button>
+              <span class="t-note t-sans search__example search__example--with-select">Examples: spending; 9003.6</span>
```
Comment (emileighoutlaw) [inline fec_eregs/templates/regulations/generic_universal.html]: Cool! Removing now

**Comment by tadhg-ohiggins** [comment]
The `develop` branch has the line https://github.com/18F/fec-eregs/blob/develop/package.json#L6 so the lowest-possible version of the `fec-style` module is installed (2.8.1), and that version has this definition of `.search__example` https://github.com/18F/fec-style/blob/7397b951c06530c843fee00f5d0ad303e0e1ef1a/scss/components/_search-bar.scss#L94

This means I see this locally even after `search__example--with-select` is removed:
image

I'm not that familiar with how `fec-style` works, but there is a much later version available as an npm module (5.8.2). Is there a reason we're keeping `fec-eregs` set to the older version?

**Comment by noahmanger** [comment]
Oh hmm. I'm not sure why it's so old. @adborden was there a reason for keeping eRegs on an old version of fec-style, or did it just never need to get updated?

**Comment by adborden** [comment]
No reason, I've only bumped it in the past when I was working on the eregs frontend, which is rare. It's not always a simple bump. Sometimes we have to reconcile new styles that clash with `regulations-site` styles.

**Comment by noahmanger** [comment]
Ok, I ended up bumping up the version and implementing these changes over in https://github.com/18F/fec-eregs/pull/319 so I'm closing this.

**Pull request closed without merging**